### PR TITLE
Fix Node example

### PR DESCRIPTION
**Pull request by rfox90**
The node connector expects the remote address to be in `options.remote` and not `address` in addition to this you need to pass the channel id in `options.channel`.

### CODE REVIEW

**Comment by ttaylorr** [comment]
:+1: /to @connor4312

File: tutorial.rst [hunk 1]
```diff
@@ -86,6 +86,8 @@ The most notable change is that, rather than trying to join a channel's chat, we
 
         return beam.game.join(stream);
     }).then(function (res) {
+        res.body.remote = res.body.address;
+        res.body.channel = stream;
         var robot = new Tetris.Robot(res.body);
```
Comment (connor4312) [inline tutorial.rst]: I'd rather pass in the options as a new object literal into the robot constructor. Modifying and shoving the request body in is a bit messy.

Comment (rfox90) [inline tutorial.rst]: Agreed. I'll get this fixed up

**Comment by rfox90** [comment]
@connor4312 Updated for you!

**Comment by connor4312** [comment]
Thanks!

**Pull request merged**